Shell/Options.cpp

```diff
@@ -1698,7 +1698,7 @@ void Options::init()
     _demodulationOnlyEquational.onlyUsefulWith(Or(_forwardDemodulation.is(notEqual(Demodulation::OFF)),_backwardDemodulation.is(notEqual(Demodulation::OFF))));
     _demodulationOnlyEquational.addProblemConstraint(hasEquality());
 
-    _extensionalityAllowPosEq = BoolOptionValue( "extensionality_allow_pos_eq","erape",false);
+    _extensionalityAllowPosEq = BoolOptionValue( "extensionality_allow_pos_eq","erape",true);
```

Comment:
While you're changing this, perhaps the unfortunate short name could be "eape"?

Reply:
A good idea! 

(Although, it also means changing all the samplers mentioning the option and my snake scripts... So I will sleep on it.)